pop-api/src/v0/fungibles/README.md

```diff
@@ -1,162 +1,19 @@
-# Fungibles API
+## Fungibles API
 
-The `fungibles` module provides an API for interacting and managing fungible tokens.
+The `fungibles` module provides an api for interacting and managing fungible tokens.
```

Comment:
https://en.wikipedia.org/wiki/API

Reply:
Changed this already while you were doing the review